modules/ppcp-compat/services.php

```diff
@@ -50,4 +52,62 @@
 		);
 	},
 
+	'compat.gzd.is_supported_plugin_version_active' => function (): bool {
+		return function_exists( 'wc_gzd_get_shipments_by_order' ); // 3.0+
+	},
+
+	'compat.gzd.tracking_statuses_map'              => function ( ContainerInterface $container ): array {
+		$gzd_statuses = wc_gzd_get_shipment_statuses();
+		$status_map = array();
+
+		foreach ( $gzd_statuses as $gzd_status ) {
+			$gzd_status = strtolower( $gzd_status );
+			switch ( $gzd_status ) {
```

Comment:
Why not just return array immediately?

```
return array(
    'draft' => 'ON_HOLD',
    ...
```

Reply:
done 7e19846c193df4637b7e5546c78b4fcc0d14347c